src/main/java/com/shaft/tools/listeners/CucumberFeatureListener.java

```diff
@@ -0,0 +1,113 @@
+package com.shaft.tools.listeners;
+
+import com.shaft.cli.FileActions;
+import com.shaft.gui.element.ElementActions;
+import com.shaft.tools.io.PropertiesFileManager;
+import com.shaft.tools.io.ReportManager;
+import io.cucumber.core.feature.FeatureParser;
+import io.cucumber.core.gherkin.Feature;
+import io.cucumber.core.resource.Resource;
+import io.cucumber.plugin.ConcurrentEventListener;
+import io.cucumber.plugin.event.*;
```

Comment:
Codacy found an issue: [Avoid unused imports such as 'io.cucumber.plugin.event'](https://app.codacy.com/manual/mohab.mohieeldeen/SHAFT_ENGINE/pullRequest?prid=5731302)

Reply:
it's used in many items below